tests/durability/test_disk_failure_recovery.py

```diff
@@ -520,3 +521,460 @@ def test_sns_repair_fail_disk_diff_cvg_equal_to_k(self):
 
         LOGGER.info("COMPLETED: Test SNS repair works fine with failed disks "
                     "are equal to K(parity units) and from different cvg")
+
+    # pylint: disable=too-many-statements
+    @pytest.mark.data_durability
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-36396")
+    def test_near_full_sns_repair_fail_disk_diff_cvg_less_than_k(self):
+        """
+        Validate SNS repair works fine on near full system with failed
+        disks are less than K(parity units)
+        """
+        LOGGER.info("STARTED: Validate SNS repair works fine on near full system with failed disks "
+                    "are less than K(parity units)")
+        near_full_percent = 60
```

Comment:
can we take from yaml in case in future we think about chnaging it?

Reply:
done